website/docs/foundations/icons/partials/design-guidelines.md

```diff
@@ -1,273 +1,149 @@
-## Design guidelines
+## Icon types
 
-### Icon types
+There are 4 types of icons: Outlined, Filled, Off, and Contained.
```

Comment:
I'm not sure if I'd consider "Off" an icon type, but not a blocker, we can discuss later